src/FffGcodeWriter.h

```diff
@@ -312,7 +312,7 @@ class FffGcodeWriter : public SettingsMessenger, NoCopy
      * \param extruder_nr The extruder for which to determine the order
      * \return A vector of mesh indices ordered on print order for that extruder.
      */
-    std::vector<unsigned int> calculateMeshOrder(const SliceDataStorage& storage, int extruder_nr) const;
+    std::vector<unsigned int> calculateMeshOrder(SliceDataStorage& storage, int extruder_nr) const;
```

Comment:
When you initialize the middle (or the aabb when you save that instead) at an earlier stage, please revert this change.